src/parser/jobs/smn/modules/SearingLight.tsx

```diff
@@ -65,17 +65,40 @@ export class SearingLight extends Analyser {
 		this.petIds = this.parser.pull.actors
 			.filter(actor => actor.owner === this.parser.actor)
 			.map(actor => actor.id)
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
```

Comment:
Not strictly part of the diff, but I'd prob one-line the pets filter too, easier to read.

Reply:
i figured i would leave the line alone since it didnt need to change, but updated per request